src/screens/DiscoveryScreenV2/fragment/SearchFragment.js

```diff
@@ -1,22 +1,20 @@
 import * as React from 'react';
-import {ScrollView, StyleSheet} from 'react-native';
+import { ScrollView, StyleSheet } from 'react-native';
 
 import DiscoveryTitleSeparator from '../elements/DiscoveryTitleSeparator';
 import RecentSearch from '../elements/RecentSearch';
-import {COLORS} from '../../../utils/theme';
+import { colors } from '../../../utils/colors';
 
-const SearchFragment = () => (
-  <ScrollView style={styles.container} keyboardShouldPersistTaps={'handled'}>
-    <RecentSearch />
-    <DiscoveryTitleSeparator text="Suggested Users" />
-  </ScrollView>
-);
+const SearchFragment = () => <ScrollView style={styles.container} keyboardShouldPersistTaps={'handled'}>
+        <RecentSearch />
+        <DiscoveryTitleSeparator text='Suggested Users' /> 
+    </ScrollView>;
 
 const styles = StyleSheet.create({
-  container: {
-    flex: 1,
-    backgroundColor: COLORS.white
-  }
-});
+    container: {
+        flex: 1,
+        backgroundColor: colors.white
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The color property value has been updated from `COLORS.white` to `colors.white`. As with the import statement, ensure that `colors.white` is equivalent to `COLORS.white` to prevent unintended UI changes.
```diff
- backgroundColor: COLORS.white
+ backgroundColor: colors.white
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->